src/api/generate-xlsx/styled-columns/index.ts

```diff
@@ -12,11 +12,15 @@ const { COLUMN_INDEXES, ADDITIONAL_COLUMN_HEIGHT } = XLSX_CONFIG;
 export const worksheetRowHeights = (worksheet: Worksheet) => {
   const modifiedWorksheet = worksheet;
 
-  modifiedWorksheet.getRow(COLUMN_INDEXES.COMPANY_ADDRESS).height = ADDITIONAL_COLUMN_HEIGHT * 2;
-  modifiedWorksheet.getRow(COLUMN_INDEXES.COMPANY_SIC_CODES).height = ADDITIONAL_COLUMN_HEIGHT;
-  modifiedWorksheet.getRow(COLUMN_INDEXES.BROKER_ADDRESS).height = ADDITIONAL_COLUMN_HEIGHT * 2;
-  modifiedWorksheet.getRow(COLUMN_INDEXES.BUYER_ADDRESS).height = ADDITIONAL_COLUMN_HEIGHT * 2;
-  modifiedWorksheet.getRow(COLUMN_INDEXES.BUYER_CONTACT_DETAILS).height = ADDITIONAL_COLUMN_HEIGHT * 2;
+  modifiedWorksheet.getRow(ROW_INDEXES.COMPANY_ADDRESS).height = ADDITIONAL_COLUMN_HEIGHT * 2;
```

Comment:
not a blocker, wonder if `2` should be a constant so all can be changed in the future if needed

Reply:
Great shout, will do this in next XLSX PR